tray/notifications.go

```diff
@@ -53,12 +33,12 @@ type dbusNotifier struct {
 func (n *dbusNotifier) start() {
 	ntf, err := newNotifier()
 	if err == nil {
-		log(pInfo, "Started dbus notifier")
```

Comment:
What is the reason for removing log prefixes?

Reply:
There was some logging change, I have added them back in next commit.